profiles/info_pkgs

```diff
@@ -29,7 +29,7 @@ sys-devel/gcc
 sys-devel/gcc-config
 dev-java/java-config
 dev-util/cmake
-dev-util/meson
+dev-build/meson
```

Comment:
These end up out-of-order but I don't think there's a point in shuffling them as cmake goes next ;-).